tests/Test.ADAL.NET.Unit/SovereignCloudsTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.ADAL.NET.Unit.Mocks;
+
+namespace Test.ADAL.NET.Unit
+{
+    [TestClass]
+    public class SovereignCloudsTests
+    {
+        private PlatformParameters platformParameters;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            HttpMessageHandlerFactory.ClearMockHandlers();
+            platformParameters = new PlatformParameters(PromptBehavior.Auto);
+        }
+
+        [TestMethod]
+        [Description("Sovereign user use world wide authority")]
+        public async Task SovereignUserWorldWideAuthorityIntegrationTest()
```

Comment:
I would encourage  you to create a separate namespace for integration tests instead of adding word integration in each test